docs/03-github/03-test-runner.mdx

```diff
@@ -287,9 +324,15 @@ jobs:
           artifactsPath: ${{ matrix.testMode }}-artifacts
           githubToken: ${{ secrets.GITHUB_TOKEN }}
           checkName: ${{ matrix.testMode }} Test Results
+          coverageOptions: 'generateAdditionalMetrics;generateHtmlReport;generateBadgeReport;assemblyFilters:+MyScripts'
```

Comment:
Is `+MyScripts` meant to be a placeholder for the users input or is this an actual token?

Perhaps we should remove it to prevent confusion? 

What is your take on this?

Reply:
Maybe we should point to the docs here concerning `assemblyFilters` https://docs.unity3d.com/Packages/com.unity.testtools.codecoverage@0.2/manual/UsingCodeCoverage.html

I'n not sure what is the default assembly name.

Reply:
There is no default assembly name, I'll add a pointer to documentation when discussing the arguments and remove the `assemblyFilters:+MyScripts` argument for now :)

Reply:
ah got it so one needs to create an assembly file for the code to be covered is that right? If so, we should probably state this as well in the docs.

Reply:
Sure, I'll add that too!